SAP-samples/fiori-elements-incident-management
# Update cds version to 7

- update cds to the latest supported version
- remove UI.Hidden from annotation file to display  table facet in OP
- update dates to 2023/2024 to see the recent data in custom section
